### ISSUE TITLE
Conan unable to install xerces-c 

### ISSUE THREAD

**Issue by professor** [greenplum-db/gpdb#3178]
I'm following the readme directions on installing GPDB on a fresh macOS.

```
Cloning into 'xerces-c'...
remote: Counting objects: 80597, done.
remote: Total 80597 (delta 0), reused 0 (delta 0), pack-reused 80597
Receiving objects: 100% (80597/80597), 24.41 MiB | 15.25 MiB/s, done.
Resolving deltas: 100% (61664/61664), done.
Note: checking out 'fda6e0a73514ac79487e91b0f171794f28979970'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

+ rm -f config.cache
+ type -p glibtoolize
+ test 1 = 0
+ libtoolize --copy --force
./reconf: line 10: libtoolize: command not found
+ aclocal -I m4
./reconf: line 13: aclocal: command not found
+ autoheader
+ automake -a -c -f
./reconf: line 15: automake: command not found
+ autocor: possibly undefined macro: AM_INIT_AUTOMAKE
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:45: error: possibly undefined macro: AM_MAINTAINER_MODE
configure.ac:69: error: possibly undefined macro: AC_LIBTOOL_WIN32_DLL
configure.ac:70: error: possibly undefined macro: AC_PROG_LIBTOOL
configure.ac:71: error: possibly undefined macro: AM_PROG_CC_C_O
configure.ac:120: error: possibly undefined macro: AC_CXX_HAVE_BOOL
configure.ac:121: error: possibly undefined macro: AC_CXX_HAVE_NAMESPACES
configure.ac:122: error: possibly undefined macro: AC_CXX_HAVE_STD_NAMESPACE
configure.ac:123: error: possibly undefined macro: AC_CXX_HAVE_STD_LIBS
configure.ac:124: error: possibly undefined macro: AC_CXX_HAVE_LSTRING
+ exit
xerces-c/Xerces-C_3_1_2@gpdb/stable: WARN: Trying to remove dirty source folder
xerces-c/Xerces-C_3_1_2@gpdb/stable: WARN: This can take a while for big packages
ERROR: xerces-c/Xerces-C_3_1_2@gpdb/stable: Error in source() method, line 15
	self.run("cd xerces-c && ./reconf")
	ConanException: Error 256 while executing cd xerces-c && ./reconf
```

**Comment by hsyuan** [comment]
patched xerces-c is not required anymore. we can just use vanilla version xerces-c from apache. But will update conan.

**Comment by hsyuan** [comment]
@pivotal-mike @oarap

**Comment by danielgustafsson** [comment]
When fixed, remember to fix the mention in the README.md file which says Conan installs Xerces.

**Comment by bhuvnesh2703** [comment]
sorry, didn't see this earlier.

The conan build process failed as some of the required packages don't exists as per the error.
```
./reconf: line 10: libtoolize: command not found
+ aclocal -I m4
./reconf: line 13: aclocal: command not found
+ autoheader
+ automake -a -c -f
./reconf: line 15: automake: command not found
```
you may install automake 1.15, autoconf 2.69 and libtoolize.

However before that the orca and xerces packages for the specific version has to be available on bintray repository. I forgot to add that noteHere is the instruction to create packages using conan: https://github.com/greenplum-db/conan

Also, i am currently working to automate pushing the orca conan packages to the bintray repository, so that they can be consumed by developers.

**Comment by bhuvnesh2703** [comment]
In order to further make it easier, the automation for building orca is on PR: https://github.com/greenplum-db/gporca/pull/237/files.

**Comment by professor** [comment]
Thanks @bhuvnesh2703. I can't think of an easy way to test the pull request from the gpdb side until the pull request is merged and https://github.com/greenplum-db/gpdb/issues/3350 is fixed.

**Comment by bchaudhary** [comment]
@professor, may be you can try running the command mentioned on the pr to build orca, as 2.46.1 orca version (latest) is there on bintray. But install xerces-c and Conan beforehand. I can pair with you to test.

**Comment by professor** [comment]
Solved by #4e9da06102a63652d58b988faf35acbad6f268d0
